docker/caffe2/jenkins/ubuntu-cuda/Dockerfile

```
@@ -28,7 +28,7 @@ RUN bash ./install_nccl.sh && rm install_nccl.sh
 # Install Python
 ARG PYTHON_VERSION
 ADD ./install_python.sh install_python.sh
-RUN if [ -n "${PYTHON_VERSION}" ]; then bash ./install_python.sh; fi
+RUN if [ -n "${PYTHON_VERSION}" ]; then bash ./install_python.sh ; fi
```

Accidental, I guess?
```suggestion
RUN if [ -n "${PYTHON_VERSION}" ]; then bash ./install_python.sh; fi
```